WordPress/Docs/NamingConventions/PrefixAllGlobalsStandard.xml

```diff
@@ -0,0 +1,67 @@
+<documentation title="Prefix All Globals">
+    <standard>
+    <![CDATA[
+      All globals terms must be prefixed with a theme/plugin specific term. Global terms include Namespace names, Class/Interface/Trait names (when not namespaced), Functions (when not namespaced or classed), Constants/Variable names declared in the global namespace, and Hook names. While prefixes must be distinct and unique to the plugin/theme, in order to prevent potential conflicts, they may be chosen by the developers and defined in a custom ruleset.
+
+https://github.com/WordPress/WordPress-Coding-Standards/wiki/Customizable-sniff-properties#naming-conventions-prefix-everything-in-the-global-namespace
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Using the prefix ECPT_">
+        <![CDATA[
+define( 'ECPT_VERSION', '1.0' );
+        ]]>
+        </code>
+        <code title="Invalid: Using a generic prefix of PLUGIN_">
+        <![CDATA[
+define( 'PLUGIN_VERSION', '1.0' );
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+      Using any prefixes reserved for WordPress are not permitted, even if WordPress has not used the define. This includes all defines such as _() (underscores), __() double underscores, wp_, and wordpress_
```

Comment:
The clarification of `(underscores)` is in parentheses, but the next clarification of `double underscores` is not. This seems inconsistent.

Worth adding `WP_` as another example, to be a bit more explicit that it's not just lowercase prefixes that are considered to be reserved?

Reply:
I've removed the second sentences as I have a feeling it would confuse people more than that it clarifies things.
Double underscores is not something checked by this sniff anyway.